site/src/components/pages/ProofMarket/FullCycle/FullCycle.tsx

```diff
@@ -11,30 +11,51 @@ import ListItem from 'components/ListItem'
 import s from './FullCycle.module.scss'
 import { homePageData } from 'stubs/homePageData'
 import { WebButton } from 'components/WebButton'
+import { Column } from 'components/Column'
 
 type FullCycleProps = {
   className?: string
   data: typeof homePageData.fullCycle
 }
 
+type ArrayElement<A> = A extends readonly (infer T)[] ? T : never
+
 const FullCycle = ({ className, data: { title, description, list, footer } }: FullCycleProps) => {
   const { isMobile } = useViewport()
+
+  const columnList = useMemo(
```

Comment:
the same.

Reply:
Merged from page branch

I will update and merge with master now